roles/network_plugin/calico/templates/kdd-crds.yml.j2

```diff
@@ -212,7 +212,11 @@ spec:
 {% endif %}
 {% if calico_version is version('v3.14.0', '>=') %}
 ---
+{% if kube_version is version('1.19.0', '>=') %}
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
I tried this but doesn't seem to pass

Reply:
Go ahead with your change, I'll rebase and package everything with my crd PR when kube-ovn release their CRD under v1